### PR TITLE
Addressing comments from IETF 103

### PR DESCRIPTION
**Pull request by richanna**
This PR addresses comments made at IETF 103:

- Error Code revisions:
  - Replaced Error Codes with smaller set of meaningfully differentiated codes.
  - Added more error response examples.
  - Added text clarifying that the value of the description attribute in error responses is implementation specific.
- Error description message languages:
  - Added requirement that error descriptions and responses are UTF-8 encoded.
  - Added error description language preferences and specification via Accept-Language and Content-Language headers.

### CODE REVIEW

File: draft-ietf-secevent-http-push.txt [hunk 1]
```diff
@@ -34,14 +34,14 @@ Status of This Memo
    Internet-Drafts are working documents of the Internet Engineering
    Task Force (IETF).  Note that other groups may also distribute
    working documents as Internet-Drafts.  The list of current Internet-
-   Drafts is at https://datatracker.ietf.org/drafts/current/.
+   Drafts is at http://datatracker.ietf.org/drafts/current/.
```
Comment (mscurtescu) [inline draft-ietf-secevent-http-push.txt]: Why switching to http? The http url redirects to the https one.

File: draft-ietf-secevent-http-push.txt [hunk 2]
```diff
@@ -53,12 +53,12 @@ Copyright Notice
...
-Backman, et al.          Expires April 21, 2019                 [Page 1]
+Backman, et al.           Expires June 9, 2019                  [Page 1]
 

-Internet-Draft        draft-ietf-secevent-http-push         October 2018
+Internet-Draft        draft-ietf-secevent-http-push        December 2018
 
 
-   (https://trustee.ietf.org/license-info) in effect on the date of
+   (http://trustee.ietf.org/license-info) in effect on the date of
```
Comment (mscurtescu) [inline draft-ietf-secevent-http-push.txt]: Same here, why http rather than https?

**Review by mscurtescu — APPROVED** [review]

File: draft-ietf-secevent-http-push.txt (hunk 1 above)
Comment (richanna) [reply draft-ietf-secevent-http-push.txt]: I was running an older version of `xml2rfc` than what Mike used to generate the current version. I've updated the PR with text and html versions generated from `xml2rfc` version 2.15.3.

File: draft-ietf-secevent-http-push.txt (hunk 2 above)
Comment (richanna) [reply draft-ietf-secevent-http-push.txt]: Fixed; see above.

**Pull request merged**